packages/components/src/components/primitives/IconButton/IconButton.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { renderWithProviders } from '@tests/renderWithProviders';
```

Comment:
Could we reuse the same import @tkanzakic ?

Reply:
done